src/components/notification/notification_event.tsx

```diff
@@ -137,7 +132,7 @@ export const EuiNotificationEvent: FunctionComponent<EuiNotificationEventProps>
         onOpenContextMenu={
           onOpenContextMenu ? () => onOpenContextMenu(id) : undefined
         }
-        eventName={eventName ?? title}
+        eventName={title}
```

Comment:
You could also always change these props from `eventName` to `title` or `eventTitle` to better align with the knowledge that that's what is expected to be passed in here.